README.md

```diff
@@ -102,14 +102,13 @@ configured cluster:
    for API, Ingress, and DNS access.
 1. The BMC IPMI addresses and credentials for the 3 control plane
    machines.
-1. Optionally, a Network Time Protocol (NTP) server where the default
-   public server is not accessible
+1. If detected that the provsioning host is not sync with a time source, configure the 25G switch as a source via the Storage DHCP and provide an optional script to set a source for the switch. 
```

Comment:
typo "provsioning" 
Change sentence to:

"If the provisioning host is not synced with a time source, configure the 25G switch to include a time source via the Storage DHCP. 

"via the Storage DHCP" doesn't sound right. What did you mean here? via the Storage network?